geode-core/src/main/java/org/apache/geode/internal/statistics/OsStatisticsProvider.java

```diff
@@ -27,47 +28,59 @@
 import org.apache.geode.internal.statistics.platform.ProcessStats;
 
 /**
- * Provides static methods which fetch operating system statistics.
- * Only Linux OS is allowed.
+ * Provides methods which fetch operating system statistics.
+ * Only Linux OS is currently allowed.
  */
-public class HostStatHelper {
-  private static final int PROCESS_STAT_FLAG = 1;
-  private static final int SYSTEM_STAT_FLAG = 2;
-  private static final boolean isLinux;
+public class OsStatisticsProvider {
+  private final int PROCESS_STAT_FLAG = 1;
```

Comment:
I think these two constants (PROCESS_STAT_FLAG and SYSTEM_STAT_FLAG) should still be static.

Reply:
What is the advantage of setting them as static? They are used only inside the OsStatisticsProvider class.

Reply:
All the instances of this class can share the same constant. You would normally only make something an instance variable if each instance needs its own copy. So the normal practice for constants is to make them static final.